.gitignore

```diff
@@ -21,3 +21,6 @@ stage/
 
 # Debug
 squashfs-root
+
+screenshot-screen.png
+screenshot-window.png
```

Comment:
Best to add comment explaining this is for the action. Otherwise someone might be confused and remove the lines again

Reply:
Fixed :)